lib/Conversion/TritonGPUToLLVM/MakeRangeOpToLLVM.cpp

```diff
@@ -25,7 +25,8 @@ struct MakeRangeOpConversion
     auto elemTy = ty.getElementType();
     assert(elemTy.isInteger(32));
     Value start = createIndexAttrConstant(rewriter, loc, elemTy, op.getStart());
-    auto idxs = emitIndices(loc, rewriter, targetInfo, layout, ty, true);
+    auto numCTAs = triton::gpu::getNumCTAs(layout);
+    auto idxs = emitIndices(loc, rewriter, targetInfo, layout, ty, numCTAs > 1);
```

Comment:
Could be an easy follow-up. It'd be nice to remove this `emitCTA` from this function, and simply always emit it, and set it to return a constant zero when `numCTAs == 1` and llvm fold everything automatically.